src/index.ts

```diff
@@ -12,7 +12,9 @@ export function createApp(targetDomain?: string) {
     domain = url.searchParams.get("domain") || "";
   }
 
-  app.setState({ domain });
+  const id = new URL(domain).searchParams.get("id") ?? "";
```

Comment:
instead of that should we have `id = url.searchParams.get("id")` in `else` clause above?